src/style.css

```diff
@@ -1 +1,98 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  background-color: white;
+  text-align: center;
+  height: 60px;
+  flex-direction: row;
+  justify-content: flex-end;
+  text-transform: uppercase;
+
+}
+
+.header {
+  display: flexbox;
+  flex-direction: column;
+  margin-bottom: 50px;
+}
+
+.link {
+  color: black;
+  text-decoration: none;
+  padding: 0 10px;
+  list-style: hover;
+  display: flex;
+  height: 40px;
+  line-height: 40px;
+}
+
+.wraper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  white-space: nowrap;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)